src/styles/components/_header.scss

```diff
@@ -1,7 +1,37 @@
+$breakpoint-tablet: 820px;
+
+// sparkbox header
 .cmp-header {
-  &__logo {
-    max-width: 12rem;
-    width: 100%;
-    height: auto;
+  padding: 1rem; // 16px
+  max-width: 100rem; // 1600px/16
+
+  // anchor tag
+  &__link {
+    outline: none;
+
+    // sparkbox logo image
+    img {
+      max-width: 12rem; // 192px/16
+      width: 100%;
+      height: auto;
+      border: 2px solid transparent;
+      transition: all 150ms ease-in-out;
+    }
+
+    &:focus img {
+      border: 2px solid var(--accent-color);
+    }
+  }
```

Comment:
Looks like here you have the `--accent-color`, but the focus for the Sparkbox logo is still the teal. Did you want it to be pink?

Reply:
Not sure why that would be as the root file specifies --accent-color to change to pink for dark mode. I'll go over this w/Noelle to make sure but it should be working.

![image](https://user-images.githubusercontent.com/49820744/199996645-7d34ce0d-ce3c-4a5c-ad83-b9192c650a78.png)

Reply:
Yeah you'll have to check with her! 
<img width="301" alt="Screen Shot 2022-11-04 at 10 41 22 AM" src="https://user-images.githubusercontent.com/69602589/200002708-4d13e2e4-7c1c-4a23-a4b6-8541c12136db.png">